src/ol/style/circle.js

```diff
@@ -107,7 +113,7 @@ ol.style.Circle = function(opt_options) {
     opacity: 1,
     rotateWithView: false,
     rotation: 0,
-    scale: 1,
+    scale: (1 / this.pixelRatio_),
```

Comment:
I don't see how it could be changed between line 102 and 112. `render_()` is called in between and it does not seems to change it neither.